src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp

```diff
@@ -57,65 +58,55 @@ MSAGeneralTab::MSAGeneralTab(MSAEditor *_msa)
     initializeParameters();
     connectSignals();
 
-    copyButton->setEnabled(msa->getUI()->getCopyFormattedSelectionAction()->isEnabled());
-
     U2WidgetStateStorage::restoreWidgetState(savableTab);
 
-#ifdef Q_OS_MAC
-    copyButton->setToolTip("Cmd+Shift+C");
-#else
-    copyButton->setToolTip("Ctrl+Shift+C");
-#endif
+    updateState();
 }
 
-void MSAGeneralTab::sl_alignmentChanged() {
-    alignmentAlphabet->setText(msa->getMaObject()->getAlphabet()->getName());
-    alignmentLength->setText(QString::number(msa->getAlignmentLen()));
-    alignmentHeight->setText(QString::number(msa->getNumSequences()));
-}
-
-void MSAGeneralTab::sl_copyFormatSelectionChanged(int index) {
-    QString selectedFormatId = copyType->itemData(index).toString();
-    emit si_copyFormatChanged(selectedFormatId);
+void MSAGeneralTab::sl_convertAlphabetButtonClicked() {
+    if (msaEditor->convertDnaToRnaAction->isEnabled()) {
+        msaEditor->convertDnaToRnaAction->trigger();
+    } else if (msaEditor->convertRnaToDnaAction->isEnabled()) {
+        msaEditor->convertRnaToDnaAction->trigger();
+    }
 }
 
-void MSAGeneralTab::sl_copyFormatted() {
-    emit si_copyFormatted();
+void MSAGeneralTab::sl_alignmentChanged() {
+    updateState();
 }
 
-void MSAGeneralTab::sl_copyFormatStatusChanged(bool enabled) {
-    copyButton->setEnabled(enabled);
+void MSAGeneralTab::sl_copyFormatSelectionChanged(int index) {
+    QString formatId = copyType->itemData(index).toString();
+    msaEditor->getUI()->getSequenceArea()->sl_changeCopyFormat(formatId);
 }
 
 void MSAGeneralTab::connectSignals() {
+    MSAEditorSequenceArea *sequenceArea = msaEditor->getUI()->getSequenceArea();
```

Comment:
Add `SAFE_POINT(sequenceArea != nullptr);`

Reply:
OptionsPanel is always initialized after main MAEditor->ui & sequence area is a required (non-null widget) which is initialized in the constructor of the MAEditor->ui